specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -7799,6 +7799,10 @@
           "description": "The target disk type after failover. Its an optional value and will be same as source disk type if not user provided.",
           "type": "string"
         },
+        "recoveryDiskEncryptionSetId": {
```

Comment:
add  new property in stable version will cause breaking change, suggest to add in new version.

Reply:
This is an optional parameter. And not a breaking change.

Reply:
These newly added properties are option (not required properties). As per PR #6380(https://github.com/Azure/azure-rest-api-specs/pull/6380/files) , it is required to bump up the API version only on adding required properties.

Reply:
@yungezz : Please add the required person , who will be doing the ARM review.